packages/common/src/index.ts

```diff
@@ -312,9 +312,8 @@ export const getFilterRuleWithDefaultValue = <T extends FilterRule>(
     ) {
         switch (filterType) {
             case FilterType.DATE: {
-                if (filterRule.operator === FilterOperator.IN_THE_PAST) {
-                    filterRuleDefaults.values =
-                        value !== undefined ? [value] : [1];
```

Comment:
This condition was here since the `in the last` filter value is a number instead of a date and has extra settings. 
Maybe what we need to change is the `else` code block. 
E.g:
```
                } else {
                    filterRuleDefaults.values = [value !== undefined ? value : new Date()];
                }
```